### PR TITLE
Add the ability to add tasks to the tasks lists.

### PR DESCRIPTION
**Pull request by ephieo**

### CODE REVIEW

File: app/controllers/app_controller.rb [hunk 1]
```diff
@@ -13,8 +13,14 @@ class TodoApp < Sinatra::Base
     erb :'home/index', :layout => :'layouts/layout'
   end
 
+  # get '/all-tasks/:id/:name' do
```
Comment (samjones1001) [inline app/controllers/app_controller.rb]: Try to remember to remove commented out code before opening a pr!

File: app/controllers/app_controller.rb [hunk 2]
```diff
@@ -13,8 +13,14 @@ class TodoApp < Sinatra::Base
     erb :'home/index', :layout => :'layouts/layout'
   end
 
+  # get '/all-tasks/:id/:name' do 
+  #   puts params
+  # end
+
   get '/all-tasks' do
     @tasks = JSON.parse(Task.get_all_tasks)
+    puts @tasks
```
Comment (samjones1001) [inline app/controllers/app_controller.rb]: I'm guessing these  puts statements were here to help with debugging? Probably also worth trying to remove these prior to opening a pr.

File: app/models/task.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 class Task < ActiveRecord::Base
+  validates :task_name, presence: true
```
Comment (samjones1001) [inline app/models/task.rb:2]: 👍

File: app/models/task.rb [hunk 4]
```diff
@@ -10,4 +12,16 @@ def self.get_all_completed_tasks
   def self.get_all_incomplete_tasks
     Task.where(completed: false).to_json
   end
+
+  def self.create_task(params)
```
Comment (samjones1001) [inline app/models/task.rb]: This looks good. This  `if model.save` pattern is super common in rails controllers

**Review by samjones1001 — APPROVED** [review]
This is a really nice PR. It's small, contains a single discreet feature, and that feature has tests associated with it - great job!

The only thing I'd pick up on is trying to remember to remove commented out code before you open PRs in the future.

File: app/controllers/app_controller.rb (hunk 1 above)
Comment (ephieo) [reply app/controllers/app_controller.rb]: 😅 👍

File: app/controllers/app_controller.rb (hunk 2 above)
Comment (ephieo) [reply app/controllers/app_controller.rb]: 👍

**Comment by ephieo** [comment]
> This is a really nice PR. It's small, contains a single discreet feature, and that feature has tests associated with it - great job!
> 
> The only thing I'd pick up on is trying to remember to remove commented out code before you open PRs in the future.

Thanks Sam !

**Pull request merged**